### PR TITLE
Swift 5 and Xcode 10.2 support

### PR DESCRIPTION
**Pull request by rynecheow**
Supporting Swift 5 and Xcode 10.2 officially

### CODE REVIEW

**Comment by freak4pc** [comment]
Hey - RxSwift 5 is coming in a bit, let’s wait for that before putting this PR together

**Comment by lukas1** [comment]
RxSwift 5 has been released :-)

**Comment by rynecheow** [comment]
Ready for review @icanzilb @freak4pc :)

**Comment by rynecheow** [comment]
Ready for review @icanzilb @freak4pc :)

File: Example/RxRealm/AppDelegate.swift [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
-import UIKit
 import RealmSwift
+import UIKit
```
Comment (freak4pc) [inline Example/RxRealm/AppDelegate.swift:1]: Why did you swap those? Not a huge deal but usually it's system-frameworks-first and then 3rd party.

File: RxRealm.podspec [hunk 2]
```diff
@@ -1,8 +1,9 @@
 Pod::Spec.new do |s|
 
   s.name             = "RxRealm"
-  s.version          = "0.7.6"
+  s.version          = "0.8.0"
```
Comment (freak4pc) [inline RxRealm.podspec]: This should be a major version bump

File: Pod/Classes/RxRealm.swift [hunk 3]
```diff
@@ -450,17 +437,16 @@ extension Reactive where Base: Realm {
...
+        -> AnyObserver<S> where S.Iterator.Element: Object {
         return AnyObserver { event in
 
             guard let elements = event.element,
                 var generator = elements.makeIterator() as S.Iterator?,
                 let first = generator.next(),
                 let realm = first.realm else {
-                    onError?(nil, RxRealmError.unknown)
-                    return
+                onError?(nil, RxRealmError.unknown)
```
Comment (freak4pc) [inline Pod/Classes/RxRealm.swift]: If this is aligned this way I think you should align the `var` and `let`s to the first (`let element`)

**Review by freak4pc — CHANGES_REQUESTED** [review]
Looks great overall! A few minor notes. I'd also like @icanzilb to review if he has the time.

**Comment by rynecheow** [comment]
Ready for another review @freak4pc

**Review by freak4pc — APPROVED** [review]
Looks good to me. Let's give @icanzilb until Monday to give this another pass. If not, I'll merge as is.

**Comment by rynecheow** [comment]
Sure @freak4pc :)

**Comment by rynecheow** [comment]
Are we ready to merge this?

**Comment by icanzilb** [comment]
I’m on vacation and the PR seems too big to review on my phone. As long as the test suite passes and the example apps (both iOS and macOS) behave as before I’m good with merging

**Comment by rynecheow** [comment]
Thanks @icanzilb! @freak4pc Would need your help to merge the PR as I can't be merging my own if all's good!

**Comment by freak4pc** [comment]
@rynecheow At home right now, I'll run it locally at night and let you know. Thanks :)

File: Cartfile.resolved [hunk 4]
```diff
@@ -1,2 +1,2 @@
-github "ReactiveX/RxSwift" "4.4.0"
-github "realm/realm-cocoa" "v3.11.1"
+github "ReactiveX/RxSwift" "5.0.0"
```
Comment (freak4pc) [inline Cartfile.resolved]: This should resolve to 5.0.1. 
5.0.0 is static.

**Review by freak4pc — APPROVED** [review]

**Pull request merged**